db/seed/dev/folders.js

```diff
@@ -0,0 +1,24 @@
+
+exports.seed = (knex, Promise) => {
+  // Deletes ALL existing entries
+  return knex('urls').del()
+    .then(() => knex('folders').del())
+    .then(() =>{
+      return Promise.all([
+        knex('folders').insert({
+          folder_name: 'default'
+        }, 'id')
+        .then(folder =>{
+          return knex('urls').insert({
+            original_url: 'www.idontknowwhattodowithmylife.com',
```

Comment:
this website is hilarious. We may the shortened URL later to reflect however we are shortening it.